src/main/docs/guide/toc.yml

```diff
@@ -108,6 +108,7 @@ httpServer:
     title: Server Side View Rendering
   openapi: OpenAPI / Swagger Support
   graphql: GraphQL Support
+  coroutines: Coroutines Support
```

Comment:
Title should maybe include "Kotlin" also I think this probably makes more sense in the `kotlin` section under `languageSupport`

Reply:
Moved to `kotlin` section.